apps/challenges/aws_utils.py

```diff
@@ -0,0 +1,542 @@
+import logging
+import os
+import random
+import string
+
+from botocore.exceptions import ClientError
+from django.conf import settings
+from http import HTTPStatus
+
+from base.utils import get_boto3_client
+
+
+logger = logging.getLogger(__name__)
+
+
+DJANGO_SETTINGS_MODULE = os.environ.get("DJANGO_SETTINGS_MODULE")
+ENV = DJANGO_SETTINGS_MODULE.split(".")[-1]
+aws_keys = {
+    "AWS_ACCOUNT_ID": os.environ.get("AWS_ACCOUNT_ID", "x"),
+    "AWS_ACCESS_KEY_ID": os.environ.get("AWS_ACCESS_KEY_ID", "x"),
+    "AWS_SECRET_ACCESS_KEY": os.environ.get("AWS_SECRET_ACCESS_KEY", "x"),
+    "AWS_REGION": os.environ.get("AWS_DEFAULT_REGION", "us-east-1"),
+}
+
+TASK_ROLE_ARN = os.environ.get("TASK_ROLE_ARN", ""),
+TASK_EXECUTION_ROLE_ARN = "arn:aws:iam::{}:role/evalaiTaskExecutionRole".format(aws_keys["AWS_ACCOUNT_ID"])
+
+DJANGO_SETTINGS_DICT = {
+    "DJANGO_SERVER": os.environ.get("DJANGO_SERVER", "localhost"),
+    "DEBUG": settings.DEBUG,
+    "EMAIL_HOST": os.environ.get("EMAIL_HOST", "https://email_host"),
```

Comment:
1. For the variables that are available in `django.conf.settings`, let's try to import them using `settings.VARIABLE_NAME` instead of `os.environ.get(VARIABLE_NAME)`. 
2. Also, let's try to not put default here since it will break stuff later on. We should raise an exception and send message to sentry if any of the following variables are missing.

Reply:
But in some cases settings file doesn't have the analogous variable for dev environment. For them, I'll stick to taking them directly as env variables with a default value, so that tests don't break.